client/lib/use-in-view/index.ts

```diff
@@ -27,20 +26,21 @@ export function useInView< T extends Element >( oneTimeCallback: () => void ): R
 				return;
 			}
 
-			oneTimeCallback();
-
+			oneTimeCallbackRef.current?.();
 			viewedRef.current = true;
+			// eslint-disable-next-line @typescript-eslint/no-use-before-define
+			observer.disconnect();
```

Comment:
We can remove this linter error if we inline the `handler` function as a parameter of the constructor instead of defining and passing the reference.

Reply:
Fixed.